xds/src/test/java/io/grpc/xds/XdsDependencyManagerTest.java

```diff
@@ -108,21 +100,20 @@ public class XdsDependencyManagerTest {
   public static final String CLUSTER_TYPE_NAME = XdsClusterResource.getInstance().typeName();
   public static final String ENDPOINT_TYPE_NAME = XdsEndpointResource.getInstance().typeName();
 
-  @Mock
-  private XdsClientMetricReporter xdsClientMetricReporter;
-
   private final SynchronizationContext syncContext =
       new SynchronizationContext((t, e) -> {
         throw new AssertionError(e);
       });
+  private final FakeClock fakeClock = new FakeClock();
+
+  private XdsClient xdsClient = XdsTestUtils.createXdsClient(
+      Collections.singletonList("control-plane"),
+      serverInfo -> new GrpcXdsTransportFactory.GrpcXdsTransport(
```

Comment:
Instead of ignoring the serverInfo, I choose to use it as the in-process server name. The SERVER_URI string previously was unused.